### DISCUSSION TITLE
Can I integrate password in my shortcut?

### DISCUSSION THREAD

**Discussion by rafoloru** [rustdesk/rustdesk#1051] [Q&A]
Hello, 

I'm using the command rustdesk.exe --connect 123456789 butI want to know if I can specify the connection password (ex: rustdesk.exe --connect ID --password)

I know it's not secure and is a clear text, bu for my need it's ok

Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/06ce3661f49e991c7d9c024b9c8f1ce07e312169

**Reply by rafoloru** [reply]
What does it mean? haha 

You already added it in the code?
